.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "hal"]
```

Comment:
I dont think this is a dev dependency.  The HAL browser is essentially the API console for humans to experiment, learn, and guide them in their scripting.
Ideally I'd like to avoid hard coding it into Spoke, as I make periodic updates to it in my github fork.
What alternatives can you think of?

Reply:
I could clone it as part of the yarn install.  what do you think about that?

Reply:
sounds great to me.  
https://github.com/opensupporter/hal-browser.git